src/OpenTelemetry/README.md

```diff
@@ -308,6 +306,19 @@ Core, Console.
 }
 ```
 
+Tip: You can find the "current working directory" by calling
+[GetCurrentDirectory](https://docs.microsoft.com/dotnet/api/system.io.directory.getcurrentdirectory)
+method from your code.
+
+As an example, for [AspNetCore example project](../../examples/AspNetCore/README.md),
```

Comment:
Do you know the case for AspNet example? (Thats where we had trouble seeing logs, so need to see if it can be documented easily, or we defer to users to find it using GetCurrentDirectory() API)

Reply:
Please see [this change](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1865) regarding asp.net.

The current working directory is not working for ASP.NET since IIS uses the `C:\Windows\System32\inetsrv` as the current directory. I have updated the logic to try to use the current directory, then the app base directory if not found.

Reply:
I've updated accordingly.